src/main/scala/ai/lum/nxmlreader/NXMLPreprocessor.scala

```diff
@@ -1,11 +1,24 @@
 package ai.lum.nxmlreader
 
-import scala.xml.{Elem, Node, Text}
+import scala.util.matching.Regex
+import scala.util.matching.Regex.Match
+import scala.xml.{ Elem, Node, Text }
 import scala.xml.transform.RewriteRule
 
 
 trait Preprocessor extends RewriteRule
 
+object PreprocessorUtils {
+
+  // We have a problem with media, in which the XML parser barfs on the inclusion of a newline
+  def preParseHook(text: String): String = {
+    def replaceNewlines(m: Match): String = m.matched.replaceAllLiterally("\n", " ")
+    val mediaFix = new Regex("<media>.*?</media>")
```

Comment:
Thanks, good catch.  The regex I had pushed looks like it will never match.  After looking into all of this further,  it seems the newlines weren't our problem.  The root cause of the issue seems to be related to how the file contents were getting read (encoding?).  I removed this pre-parse hook and tested again with a sample of 1K OA papers.  No errors!